.storybook/main.js

```diff
@@ -3,13 +3,7 @@ const { propNames } = require("@chakra-ui/react")
 const { babelConfig } = require("./babel-storybook-config")
 
 module.exports = {
-  stories: [
-    {
-      directory: "../src/components",
-      titlePrefix: "Components",
-      files: "**/*.stories.tsx",
-    },
-  ],
+  stories: ["../src/components/**/*.stories.tsx"],
```

Comment:
This changed can be placed in a separate PR if preferred (not going to impact title in the story file of this PR)